pom.xml

```diff
@@ -157,19 +167,31 @@
         <version>3.1.0</version>
         <configuration>
           <archive>
+            <!-- Pull in the bnd generated manifest -->
+            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
             <manifest>
               <mainClass>org.bytedeco.javacpp.tools.Builder</mainClass>
             </manifest>
+            <!-- Multi-Release: true goes in the main attributes -->
             <manifestEntries>
-              <Name>org/bytedeco/javacpp/</Name>
-              <Implementation-Title>${project.name}</Implementation-Title>
-              <Implementation-Vendor>Bytedeco</Implementation-Vendor>
-              <Implementation-Version>${project.version}</Implementation-Version>
-              <Specification-Title>${project.name}</Specification-Title>
-              <Specification-Vendor>Bytedeco</Specification-Vendor>
-              <Specification-Version>${project.version}</Specification-Version>
               <Multi-Release>true</Multi-Release>
             </manifestEntries>
+            <!-- 
+              We must use a Manifest Section here so that the
+              maven-jar-plugin merges the manifest metadata
+              correctly rather than jumbling things up
+            -->
+            <manifestSection>
+              <name>org/bytedeco/javacpp/</name>
+              <manifestEntries>
+                <Implementation-Title>${project.name}</Implementation-Title>
+                <Implementation-Vendor>Bytedeco</Implementation-Vendor>
+                <Implementation-Version>${project.version}</Implementation-Version>
+                <Specification-Title>${project.name}</Specification-Title>
+                <Specification-Vendor>Bytedeco</Specification-Vendor>
+                <Specification-Version>${project.version}</Specification-Version>
+              </manifestEntries>
+            </manifestSection>
```

Comment:
It looks like because of this change this is not able to pick up the version number anymore:
```java
String version = Builder.class.getPackage().getImplementationVersion();
```
https://github.com/bytedeco/javacpp/blob/master/src/main/java/org/bytedeco/javacpp/tools/Builder.java#L1214
Any ideas why that might be?

Reply:
So my intention with this was to maintain the correct the layout in the manifest. The existing intstructions include a `Name` header, which is a reserved header designed to tie the next header(s) to a specific package. Unfortunately when merging these values into an existing manifest the maven-jar-plugin was inserting the entries at random points and therefore resulting in a broken Manifest. Apparently I got far enough to make the manifest valid, but not quite all the way to making it pick up the package-specific values.

Reply:
> Any ideas why that might be?

I've just figured this out - the `Name` header is limiting the following:
```
Specification-Title: JavaCPP
Specification-Vendor: Bytedeco
Implementation-Vendor: Bytedeco
Specification-Version: 1.5.2-SNAPSHOT
Implementation-Version: 1.5.2-SNAPSHOT
Implementation-Title: JavaCPP
```
 to only apply to the `org.bytedeco.javacpp` package, which I assume was the original intent of having the named section. This PR has now "fixed" this to work properly, rather than getting inconsistent behaviour from the maven-jar-plugin. 

The bug comes in because the Builder is attempting to get the version of its own package, which is `org.bytedeco.javacpp.tools` not `org.bytedeco.javacpp`. This package has no section in the manifest and so is unversioned.

There are three possible fixes I can think of:
 1. Change the named section to point at the tools package
 2. Add a new named section for the tools package
 2. Remove the name from the section and have the manifest apply to all packages

Any preferences?

Reply:
I'd rather just leave it like it was before (all packages). It doesn't matter if the entries in the manifest file don't end up in any meaningful order. They never do anyway and we don't often look at it manually either.